### PR TITLE
Gridlayer tweaks

### PR DESCRIPTION
**Pull request by sigvef**
http://i.gyazo.com/3934962ec885270ef6cb3c66c371d375.gif (worlds biggest gif :3, turns out noise fx is bad for compression)

Small evolution of the grid

### CODE REVIEW

File: res/layers.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "type": "NoiseFXLayer",
     "displayName": "Noise fx",
     "startFrame": 654,
-    "endFrame": 1443,
+    "endFrame": 1716,
     "color": "#521b9f"
   }
-]
+]
```
Comment (cristeahub) [inline res/layers.json:34]: Is this caused by moving stuff in nin? If so we should fix it.

Comment (sigvef) [inline res/layers.json:34]: Probably. Go fix :P

**Comment by aleksanb** [comment]
Linewidth 2 looks much better than 1.
If windows allways sets it to 1, can it just be left at 2?

**Comment by stianjensen** [comment]
> If windows allways sets it to 1, can it just be left at 2?

Then it won't look the same in all browsers

**Comment by sigvef** [comment]
> Why BEAN and not relativeframe?

To sync with music changes

**Comment by aleksanb** [comment]
Needs rebase

**Comment by aleksanb** [comment]
Closing. this is contained in the oterh one.

**Pull request closed without merging**